front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -254,32 +253,19 @@ export const stdcmConfSlice = createSlice({
       );
       state.stdcmPathSteps = newPathSteps;
     },
-    addStdcmSimulation(
-      state: Draft<OsrdStdcmConfState>,
-      action: PayloadAction<StdcmSimulationInputs>
-    ) {
+    addStdcmSimulation(state: Draft<OsrdStdcmConfState>, action: PayloadAction<StdcmSimulation>) {
```

Comment:
Hm maybe it would be nice to modify this function to add several simulations at the time
```suggestion
    addStdcmSimulations(state: Draft<OsrdStdcmConfState>, action: PayloadAction<StdcmSimulation[]>) {
```